komorebi.in

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import gi
+import signal
+import sys
+import logging
+
+gi.require_versions({
+	'Gtk': '3.0',
+	'GObject': '2.0',
+	'GtkClutter': '1.0',
+	'Clutter': '1.0',
+	'ClutterGst': '3.0',
+	'Gdk': '3.0',
+	'GdkPixbuf': '2.0',
+	'Gst': '1.0',
+	'WebKit2': '4.0',
+})
+
+from komorebi.preferences_window import PreferencesWindow, SETTINGS_CHANGED_SIGNAL
```

Comment:
I don't really like partial import, especially in code I maintain, but if you want to use them it's fine, this is just a FYI.